### PR TITLE
Update .env.example

### PR DESCRIPTION
**Pull request by Xteri08**
Improved Arrangement and added more description for the config file.

### CODE REVIEW

**Comment by Xteri08** [comment]
PING_PROXY_SERVER_USING_IP_ADDRESS
Is enabled by default

**Comment by Xinecraft** [comment]
Plz add space after each comment hash (#).

Eg:
```
# Laravel Config
# Dont enable on production
```

File: .env.example [hunk 1]
```diff
@@ -1,77 +1,58 @@
+#Laravel Config
```
Comment (Xinecraft) [inline .env.example:1]:
```
# Laravel Config
```

File: .env.example [hunk 2]
```diff
@@ -1,77 +1,58 @@
...
 APP_NAME=Minetrax
 APP_ENV=local
 APP_KEY=
+#Set your websuite language by default
```
Comment (Xinecraft) [inline .env.example:5]:
```
# Set your websuite language by default
```
same for all below...

**Review by Xinecraft — CHANGES_REQUESTED** [review]

**Comment by Xteri08** [comment]
> Plz add space after each comment hash (#).
> 
> Eg:
> ```
> # Laravel Config
> # Dont enable on production
> ```

Give me a movement I forget that

**Pull request merged**